webapp/src/Service/SubmissionService.php

```diff
@@ -282,6 +282,12 @@ public function getSubmissionList(
             ->andWhere($countQueryExtras['queued'])
             ->getQuery()
             ->getSingleScalarResult();
+        $counts['inContest'] = (clone $queryBuilder)
+            ->select('COUNT(s.submitid)')
+            ->join('s.contest', 'c')
+            ->andWhere('s.submittime BETWEEN c.starttime AND c.endtime')
```

Comment:
I think we also want to filter out submissions from teams in invisible categories?